app/src/main/java/org/jellyfin/androidtv/ui/itemhandling/ItemLauncher.java

```diff
@@ -137,8 +138,16 @@ public static void launch(final BaseRowItem rowItem, ItemRowAdapter adapter, int
                         return;
 
                     case Audio:
-                        //produce item menu
-                        KeyProcessor.HandleKey(KeyEvent.KEYCODE_MENU, rowItem, activity);
+                        Timber.d("got pos %s", pos);
+                        // if a song isn't the first in the queue, play it
+                        if (rowItem.getIndex() > KoinJavaComponent.<MediaManager>get(MediaManager.class).getCurrentAudioQueuePosition() && rowItem.getIndex() < KoinJavaComponent.<MediaManager>get(MediaManager.class).getCurrentAudioQueueSize()) {
+                            KoinJavaComponent.<MediaManager>get(MediaManager.class).playFrom(rowItem.getIndex());
+                        } else {
+                            // otherwise, open AudioNowPlayingActivity
+                            Intent nowPlaying = new Intent(activity, AudioNowPlayingActivity.class);
+                            activity.startActivity(nowPlaying);
+                        }
+
```

Comment:
I get this crash when I press an album/track card. For example: an item in the "latest music" section in the home screen or an item in "last played" in the smart screen for music libraries.

```
D/ItemLauncher: got pos 0
I/AppThemeCallbacks: Applying theme: DARK
D/CompatibilityChangeReporter: Compat change id reported: 171228096; UID 10077; state: ENABLED
I/BackgroundService: Adding BackgroundServiceFragment to activity
D/AndroidRuntime: Shutting down VM
    
    
    --------- beginning of crash
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.jellyfin.androidtv.debug, PID: 5861
    java.lang.RuntimeException: Unable to start activity ComponentInfo{org.jellyfin.androidtv.debug/org.jellyfin.androidtv.ui.playback.AudioNowPlayingActivity}: java.lang.IllegalArgumentException: ObjectAdapter cannot be null
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3635)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3792)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2210)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loopOnce(Looper.java:201)
        at android.os.Looper.loop(Looper.java:288)
        at android.app.ActivityThread.main(ActivityThread.java:7839)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
     Caused by: java.lang.IllegalArgumentException: ObjectAdapter cannot be null
        at androidx.leanback.widget.ListRow.verify(ListRow.java:51)
        at androidx.leanback.widget.ListRow.<init>(ListRow.java:34)
        at org.jellyfin.androidtv.ui.playback.AudioNowPlayingActivity.addQueue(AudioNowPlayingActivity.java:281)
        at org.jellyfin.androidtv.ui.playback.AudioNowPlayingActivity.onCreate(AudioNowPlayingActivity.java:275)
        at android.app.Activity.performCreate(Activity.java:8051)
        at android.app.Activity.performCreate(Activity.java:8031)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1329)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3608)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3792) 
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103) 
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2210) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loopOnce(Looper.java:201) 
        at android.os.Looper.loop(Looper.java:288) 
        at android.app.ActivityThread.main(ActivityThread.java:7839) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003) 
E/ACRA: ACRA caught a RuntimeException for org.jellyfin.androidtv.debug
    java.lang.RuntimeException: Unable to start activity ComponentInfo{org.jellyfin.androidtv.debug/org.jellyfin.androidtv.ui.playback.AudioNowPlayingActivity}: java.lang.IllegalArgumentException: ObjectAdapter cannot be null
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3635)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3792)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2210)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loopOnce(Looper.java:201)
        at android.os.Looper.loop(Looper.java:288)
        at android.app.ActivityThread.main(ActivityThread.java:7839)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003)
     Caused by: java.lang.IllegalArgumentException: ObjectAdapter cannot be null
        at androidx.leanback.widget.ListRow.verify(ListRow.java:51)
        at androidx.leanback.widget.ListRow.<init>(ListRow.java:34)
        at org.jellyfin.androidtv.ui.playback.AudioNowPlayingActivity.addQueue(AudioNowPlayingActivity.java:281)
        at org.jellyfin.androidtv.ui.playback.AudioNowPlayingActivity.onCreate(AudioNowPlayingActivity.java:275)
        at android.app.Activity.performCreate(Activity.java:8051)
        at android.app.Activity.performCreate(Activity.java:8031)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1329)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3608)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3792) 
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103) 
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2210) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loopOnce(Looper.java:201) 
        at android.os.Looper.loop(Looper.java:288) 
        at android.app.ActivityThread.main(ActivityThread.java:7839) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1003) 
D/CompatibilityChangeReporter: Compat change id reported: 149924527; UID 10077; state: ENABLED
D/CompatibilityChangeReporter: Compat change id reported: 132649864; UID 10077; state: DISABLED
I/Process: Sending signal. PID: 5861 SIG: 9
```

Reply:
I just saw this. I'm looking into it